src/scenes/AcceleratorRouter/Overview/index.tsx

```diff
@@ -1,15 +1,95 @@
-import React from 'react';
+import { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 
+import { Grid, useTheme } from '@mui/material';
+
 import Page from 'src/components/Page';
 import Button from 'src/components/common/button/Button';
 import { APP_PATHS } from 'src/constants';
 import strings from 'src/strings';
+import { Scorecard } from 'src/types/Score';
 import useDeviceInfo from 'src/utils/useDeviceInfo';
 
+import ScoreEntry from '../Scoring/ScoreEntry';
+
+const mockQualitativeInfo =
```

Comment:
This is a hacky demo to test the component, will remove all changes in this scene before merging.